app/controllers/sites/redesigned_displays_controller.rb

```diff
@@ -3,5 +3,24 @@
 class Sites::RedesignedDisplaysController < Sites::SetupSiteController
   def edit; end
 
-  def update; end
+  def update
+    if @site.update(site_params)
+      redirect_to edit_site_redesigned_display_path(@site),
+                  flash: { success: 'You have updated your font & colors.' }
+    else
+      render :edit
```

Comment:
This is probably not an entirely bad suggestion, but we currently don't provide any flash messages in this scenario in any of the other `app/controllers/sites` controllers at present, so this might be a new visual change that isn't expected and wasn't requested.  I'd rather keep the implementation basic for now and enhance if/when needed.  As it stands, it's exceedingly difficult to trigger this scenario in the UI since "Save" is inactive unless updated params are provided.  Also, in the case of invalid fields, a flash message will be displayed as a result of the validation failure.